entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -284,7 +284,7 @@ class EntityHandler(
                 AttributeReplaceEvent(
                     entityId.toUri(),
                     compactTerm(updatedDetails.attributeName, contexts),
-                    extractDatasetIdFromNgsiLdAttributes(ngsiLdAttributes, updatedDetails.attributeName),
```

Comment:
is `extractDatasetIdFromNgsiLdAttributes` still used elsewhere? if not, remove it.

Reply:
Indeed, removed.